src/test/integration/utils/mochi.js

```diff
@@ -0,0 +1,52 @@
+
+function info(msg) {
+  dump(`info: ${msg}\n`);
+}
+
+var ContentTask, gBrowser, openNewTabAndToolbox, Services, EXAMPLE_URL;
```

Comment:
I don't love this DI style, we hopefully can get this file priviledged like `head.js` with the proper permissions